site/src/pages/TerminalPage/TerminalPage.tsx

```diff
@@ -166,9 +167,13 @@ const TerminalPage: FC<TerminalPageProps> = ({ renderer }) => {
         background: colors.gray[16],
       },
     });
-    // DOM is the default renderer.
-    if (renderer === "webgl") {
+    if (config.data && config.data.config.web_terminal_renderer === "webgl") {
```

Comment:
Not important or anything (if we really care we should use a linting rule), but just a note that we seem to generally prefer `?` in this situation:

```suggestion
    if (config.data?.config.web_terminal_renderer === "webgl") {
```